HW1/P2/P2b.py

```diff
@@ -1,3 +1,26 @@
 from P2 import *
 
-# Your code here
+import pyspark
+
+from P2 import *
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+sc =pyspark.SparkContext()
+
+data=[]
+for i in range(0,1999):
+    for j in range(0,1999):
+      data.append([i,j])
+
+#print data
+rdd=sc.parallelize(data)
+rdd=rdd.repartition(100)
```

Comment:
Good use of repartition. Note that the spread is 1,400,000 to 2,700,000, which is very good. There are ways to make this spread even smaller (and therefore more equally sized tasks), by using a uniformly random distribution as a hash function.